### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by HazemElknawy9** [giuliopons/adadmin#112] [General]
when added banner from account  advertiser i have this error
Fatal error: Uncaught Error: Undefined constant "COINBASE_API_KEY" in E:\PC\laragon\www\construction\ads\src\componenti\7banner\_include\banner.class.php:787 Stack trace: #0 E:\PC\laragon\www\construction\ads\src\componenti\7banner\index.php(395): Banner->checkoutForm(81) #1 {main} thrown in E:\PC\laragon\www\construction\ads\src\componenti\7banner\_include\banner.class.php on line 787

**Comment by giuliopons** [comment]
Strange, this should not happen.
Try to execute again the installation by going on the address:
https://www.yourdomain.com/amb/src/componenti/install

(maybe you have "ads" instead of "amb")

**Comment by HazemElknawy9** [comment]
Yes I used ads instead of amb Is this problem?

في الأحد، ١٩ مايو ٢٠٢٤، ٤:٤٧ م Giulio Pons ***@***.***> كتب:

**Comment by giuliopons** [comment]
No, that's not a problem.

Il dom 19 mag 2024, 16:44 HazemElknawy ***@***.***> ha
scritto:
